Swap Fee Parameter updates (#137)
NethermindEth/Forta-Agents: Impossible-Agent-Suite/SwapFees/README.md

# Impossible Finance - Swap Fee Parameter Updates 

## Description

This agent detects transactions where modifications in fee values of Impossible Finance V2 Pairs occurred.
> Check Impossible Finance V2 Pairs and Factory here https://github.com/ImpossibleFinance/impossible-swap-core

> 

Reviewer: ```suggestion
> You can update the Factory address to V2 Factory by changing `FACTORY` value in the agent file. 

```

## Supported Chains

- BSC

## Alerts

- impossible-6-1

>   - Fired when a `UpdatedTradeFees` events are emitted in Impossible Finance V2 Pairs

Reviewer 2: ```suggestion
  - Fired when an `UpdatedTradeFees` event is emitted by an Impossible Finance V2 pair
```

  - Severity is always set to "Info" 
  - Type is always set to "Info" 
  - Metadata contains:
    - `pair`: Address of the pair emitting the event
    - `oldFee`: Old trade fee value
    - `newFee`: New trade fee value

- impossible-6-2

>   - Fired when a `UpdatedWithdrawalFeeRatio` events are emitted in Impossible Finance V2 Pairs

Reviewer 2: ```suggestion
  - Fired when an `UpdatedWithdrawalFeeRatio` event is emitted by an Impossible Finance V2 pair
```
